index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <header class="app-header">
+        <a href="#" class="logo">Narson</a>
+        <nav class="ambony">
+            <a href="#"><img src="media/Union.svg" alt="humburger" class="humburger"></a>
+            <ul class="menu">
+                <li class="menu-item"><a href="#">Portfolio</a></li>
+                <li class="menu"><a href="#">About</a></li>
+                <li class="menu"><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class="header-ambony">
+        <h1 class="narson">I'm Narson.
+            <br>Glad to see you!
+        </h1>
+        <p class="detail">
+            I'm a software developer! I can 
+            help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <a href="#" class="connection"><h4>LET'S CONNECT</h4></a>
+
+        <ul class="social">
+            <a href="https://twitter.com/@Narson321"><li class="twitter">
+                <img src="media/twitter.svg" alt="twitter" title="twitter"/>
+            </li></a>
+            <a href="https://linkedin.com/in/andry-narson-rabedesana-15b8b4248"><li class="linkindin">
+                <img src="media/Linkedin icon.svg" alt="Linkindin" title="linkindin"/>
+            </li></a>
+            <a href="https://github.com/rbd3"><li class="github">
+                <img src="media/github.svg" alt="github" title="github" />
+            </li></a>
+            <a href="#"><li class="angellist">
+                <img src="media/angellist.svg" alt="Angellist" title="Angellist" />
+            </li></a>
+            <a href="#"><li class="medium">
+                <img src="media/medium.svg" alt="Medium" title="Medium" />
+            </li></a>
+        </ul>
```

Comment:
- [ ] As asked by the previous reviewer, kindly adjust the social links to that of the Figma design. The given color is `background: #505F79;`. This has to be specific 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/228450777-3ddb7b33-d1b4-497a-bb68-5cd84da4d4d5.png)

## figma:
![image](https://user-images.githubusercontent.com/50990925/228450907-450e97cc-0092-4014-a993-cfcc095bd5a0.png)

Reply:
hi @Shedrack-Sunday ,
I don't understand what you asking me to do. I already change my social color here '.social {
  color: #505f79;
} '

Reply:
@Shedrack-Sunday 
Do you see my question above?

Reply:
@Shedrack-Sunday 
I changed the color of my social icons image like this ' .twitter, .linkindin, .github, .medium, .angellist { color: #505f79;}. But I don't see a change on my page.